### ISSUE TITLE
notification_action_cable_allowed? error

### ISSUE THREAD

**Issue by szechyjs** [simukappu/activity_notification#138]
### Steps to reproduce
Setup up notifications so that `User` receives notifications for `User` create.

```rb
acts_as_notifiable :users,
  targets: ->(user, key) {
    User.with_role(:admin)
  },
  tracked: { only: [:create] },
  notifiable_path: :user_notifiable_path,
  printable_name: ->(user) { "User #{user.email}" }
```

Open the user's notifications view.

### Expected behavior
Should render the page without error

### Actual behavior
The following error:
```
ActionView::Template::Error (wrong number of arguments (given 0, expected 1..2)):
    39:   }
    40: </style>
    41:
    42: <% if @target.notification_action_cable_allowed? %>
    43:   <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/push.js/1.0.9/push.min.js"></script>
    44:   <script>
    45:     App.activity_notification = App.cable.subscriptions.create(

activity_notification (9f35a5b3a420) lib/activity_notification/models/concerns/notifiable.rb:165:in `notification_action_cable_allowed?'
activity_notification (9f35a5b3a420) app/views/activity_notification/notifications/default/index.html.erb:42
```

### System configuration
**activity_notification gem version**: 2.1.2 (master)
**Rails version**: 6.0.3
**ORM (ActiveRecord, Mongoid or Dynamoid)**: ActiveRecord

**Comment by simukappu** [comment]
Thank you for your report.
The reason of this issue is that we have two different method *ActivityNotification::Target#notification_action_cable_allowed?(notifiable = nil, key = nil)* and *ActivityNotification::Notifiable#notification_action_cable_allowed?(target, key = nil)*.
If you could customize your view, we would appreciate your pull request.

**Comment by simukappu** [comment]
We've just published as v2.1.4.
https://rubygems.org/gems/activity_notification/versions/2.1.4

Thank you!
